django/db/models/sql/where.py

```diff
@@ -183,6 +222,14 @@ def relabeled_clone(self, change_map):
         clone.relabel_aliases(change_map)
         return clone
 
+    def replace_expressions(self, replacements):
+        if replacement := replacements.get(self):
+            return replacement
```

Comment:
Is this possible? `coverage` shows that it's untested.

Reply:
Added an assertion to cover it; a `Q` annotation referring to a window function that is later filtered against will cover this branch.